qiita_db/test/test_processing_job.py

```diff
@@ -256,6 +258,15 @@ def test_create(self):
         self.assertEqual(obs.step, None)
         self.assertTrue(obs in qdb.artifact.Artifact(1).jobs())
 
+    def test_create_duplicated(self):
+        job = _create_job()
+        job._set_status('running')
+        with self.assertRaisesRegexp(ValueError, 'Cannot create job because '
+                                     'the parameters are the same as jobs '
+                                     'that are queued, running or already '
+                                     'have succeeded:'):
```

Comment:
How is this passing if this is not a regex, and there's no pattern to match here? Shouldn't there be a job id, or a regex indicating that this should match a job id?

Reply:
cause it behaves as an IN.

Reply:
Got it!